Ryan/more podman documentation (#2267)
earthly/earthly: docs/guides/podman.md

# or
> yum install qemu-user-static
```

### crun: open executable: Permission denied: OCI permission denied

> This can happen when you are attempting to run a binary without the execution permission; for example, as part of an ENTRYPOINT command.

Reviewer: ```suggestion
This can happen if you attempt to run (or the `ENTRYPOINT` references) a binary without the execution permission.
```

> https://github.com/containers/podman/issues/9377

Reviewer: should these be here? or should they be markdown links?
